server/src/db/migrations/20220503184204_createReviews.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("reviews", (table) => {
```

Comment:
This file appears to have a tab size of 4 spaces, while other files in the app have a tab size of 2. This should be consistent for all group members, you can change this setting in VSCode to set the `tab size` of your editor to 2!